src/api/spec/factories/project.rb

```diff
@@ -3,6 +3,14 @@
     sequence(:name) { |n| "project_#{n}" }
     title { Faker::Book.title }
 
+    after(:create) do |project|
+      # NOTE: Enable global write through when writing new VCR cassetes.
+      # ensure the backend knows the project
+      if CONFIG['global_write_through']
+        Suse::Backend.put("/source/#{CGI.escape(project.name)}/_meta", project.to_axml)
```

Comment:
But that's a different topic anyway.

And since this is only about setting up test data, I think it's ok to use the direct backend call.